content/blog/2025/04/2025-04-02-matrix-auth-service.md

```diff
@@ -46,24 +47,25 @@ MAS comes with a significant refactoring of how authentication works on Matrix.
 * Restricted access to sensitive operations, like deactivating your account.
 * Clearer view of which clients are using your account.
 
-<div style="display: grid;place-content:center;">
-<img src="/blog/img/mas-grant-access.webp" />
-</div>
+{{ figure(
+    img="/blog/img/mas-grant-access.webp"
+    caption="An example of the scope request view of MAS showing Element requesting permissions to see the profile, view existing messages and data and sending messages on your behalf.")
+}}
 
 ### Improved registration experience
 
 Regardless of the client you use, the new registration and login experience makes it clear where your account lives, and it supports next-generation clients like Element X.
 
-<div style="display: grid;place-content:center;">
-<img src="/blog/img/mas-create-account.webp" />
-</div>
-
+{{ figure(
+    img="/blog/img/mas-create-account.webp"
+    caption="The new Registration Dialog for MAS showing an input field for a username and various social logins")
+}}
 
 ## Impact
 
 Your current sessions will remain active after the migration has taken effect. In other words, you will not be logged out of your clients.
 
-We’re providing backwards compatibility for existing Matrix clients - this does not remove the stable Matrix 1.0 APIs. You can read more about the impact on clients in our previously published blog article - [Authentication changes on Matrix.org](https://matrix.org/blog/2025/01/06/authentication-changes/).
+We’re providing backwards compatibility for existing Matrix clients - this does not remove the stable Matrix 1.14 APIs. You can read more about the impact on clients in our previously published blog article - [Authentication changes on Matrix.org](https://matrix.org/blog/2025/01/06/authentication-changes/).
```

Comment:
Ideally per our policy align with Quentin (credited author) here, I can image what was intended by saying "Matrix 1.0" despite maybe not being correct.
Another way to phrase this if difficult to cover the technically correct versions could be something with "pre-Matrix 2.0".

Reply:
Tbh I would prefer if the foundation is factual rather than contradicting itself. Matrix 2.0 already had caused plenty and probably forever existing confusion across the less techy people of matrix and I rather would avoid having another instance but on the reverse. Especially as all of the 2.0 features are landing in a 1.x release of spec. Making 1.0 even more confusing as a term. Especially as there never existed a 1.0 spec.

Reply:
I like the "pre-Matrix 2.0" workaround